apps/web/src/components/search/search-content.tsx

```diff
@@ -0,0 +1,196 @@
+import { DialogClose } from '@radix-ui/react-dialog';
+import type { Challenge } from '@repo/db/types';
+import { DifficultyBadge } from '@repo/ui/components/difficulty-badge';
+import Link from 'next/link';
+import { Highlight } from 'react-instantsearch';
+import { SearchBoxContextProvider, useSearchBox } from './searchbox.context';
+import { useSearchResult, useSearchStatus, type Result } from './search-provider';
+import { useRecentSearchesStorage } from './use-recent-searches-storage';
+import { Text } from '@repo/ui/components/typography/typography';
+import { SearchIcon, Loader2 as LoaderIcon, X as XIcon } from '@repo/ui/icons';
+import { ScrollArea, ScrollBar } from '@repo/ui/components/scroll-area';
+
+function useRecentSearches() {
+  return useRecentSearchesStorage<Challenge>();
+}
+
+type OnClick = () => void;
+
+interface HitsProps {
+  onClick: OnClick;
+}
+
+function Results({ onClick }: HitsProps) {
+  const { query, results } = useSearchResult();
+
+  if (!query) {
+    return (
+      <div className="mb-6 flex flex-col">
+        <RecentSearches onClick={onClick} />
+      </div>
+    );
+  }
+
+  if (results.length === 0) {
+    return (
+      <div className="flex h-full flex-grow flex-col justify-between">
+        <Text intent="leading" className="my-8 flex items-center justify-center">
+          No results found for "<strong className="max-w-[50%] md:max-w-[400px]">{query}</strong>"
+        </Text>
+        <ProposedPhrases />
+      </div>
+    );
+  }
+
+  return (
+    <div className="mb-6 flex flex-col">
+      {results.map((result) => (
+        <Result key={result.id} result={result} onClick={onClick} />
+      ))}
+    </div>
+  );
+}
+
+function Result({ result, onClick }: { result: Result; onClick: OnClick }) {
+  const { onAdd } = useRecentSearches();
+  if (result.status !== 'ACTIVE') return null;
+
+  return (
+    <Link
+      prefetch={false}
```

Comment:
is there a prefetch on hover option for link

Reply:
So you're suggesting to add prefetching on hover?

Reply:
no it’s fine. was hoping it had that with some option